### DISCUSSION TITLE
Set character/Vehicle selection based on world selection?

### DISCUSSION THREAD

**Discussion by DaPiMan** [Berny23/LD-ToyPad-Emulator#16] [Ideas]
An idea, I see in the charactermap.json we have "world" data; would it be possible to populate the character/vehicle selection list from a prior world selection?

I'm really not familiar with .json at all, I had a tinker and got the list of worlds but couldn't base the list of characters from that world selection. I think it would be a really good upgrade as the "worlds" are really the packs that were bought in the stores and it would give quick access to all the characters/vehicles from that particular set.

I'll keep tinkering to see if I can get it working but I'm really a noob when it comes to this :)

**Comment by Berny23** [comment]
I don't have much time lately, so feel free to try it.
A selection at the top could filter the character and vehicle selection dropdown fields according to the world. Like "All Worlds, World A, World B, ..."

Just look at my code for getting the other JSON data. I'm sure there is a filter function or library for dropdown fields.

A quick note: Images of the worlds/packs/characters/vehicles would be nice, but I strongly advice against it, because I don't want this repo to get a DMCA takedown for copyrighted logos or character art.

I'm thankful for your contributions!
